src/psyclone/tests/domain/lfric/dofkern_test.py

```diff
@@ -341,7 +321,8 @@ def test_multi_invoke_cell_dof_builtin(tmpdir, monkeypatch, annexed, dist_mem):
             output = (
                 "      DO df = loop0_start, loop0_stop, 1\n"
                 "        CALL testkern_dofs_code(f1_data(df), f2_data(df), "
-                "f3_data(df), f4_data(df), scalar_arg)\n"
+                "f3_data(df), f4_data(df), scalar_arg, field_vec_1_data, "
+                "field_vec_2_data, field_vec_3_data)\n"
                 "      END DO\n"
                 "      !\n"
                 "      ! Set halos dirty/clean for fields modified in the "
```

Comment:
I would guess this test does not pass when compilation is enabled (`pytest --compile domain/lfric/dofkern_test.py`)?